### PR TITLE
Add support handling deeplink

### PR DESCRIPTION
**Pull request by vinothvino42**
Added missing support to handle deeplink with passed-in URL

Another PR (ensemble_deeplink) - https://github.com/EnsembleUI/ensemble_deeplink/pull/1

### CODE REVIEW

File: lib/action/deep_link_action.dart [hunk 1]
```diff
@@ -70,6 +70,58 @@ class DeepLinkInitAction extends EnsembleAction {
...
+      );
+    }
+    throw LanguageError('DeferredDeepLink: Missing inputs for handleDeepLink');
+  }
+
+  @override
+  Future execute(BuildContext context, ScopeManager scopeManager) async {
+    try {
+      GetIt.I<DeferredLinkManager>().handleDeferredLink(url!, (linkData) {
+        return ScreenController().executeAction(context, onLinkReceived!,
```
Comment (vusters) [inline lib/action/deep_link_action.dart:113]: null check

File: lib/action/deep_link_action.dart [hunk 2]
```diff
@@ -70,6 +70,58 @@ class DeepLinkInitAction extends EnsembleAction {
...
+            event: EnsembleEvent(initiator, data: {'link': linkData}));
+      });
+    } catch (e) {
+      return ScreenController().executeAction(context, onError!,
```
Comment (vusters) [inline lib/action/deep_link_action.dart:117]: null check

**Review by vusters — CHANGES_REQUESTED** [review]

File: lib/action/deep_link_action.dart (hunk 1 above)
Comment (vinothvino42) [reply lib/action/deep_link_action.dart]: Fixed

File: lib/action/deep_link_action.dart (hunk 2 above)
Comment (vinothvino42) [reply lib/action/deep_link_action.dart]: Fixed

**Review by vusters — APPROVED** [review]

**Pull request merged**